pages/blog/_slug.vue

```diff
@@ -2,20 +2,21 @@
   <div>
     <div class="container position-relative">
       <article
+        v-if="blogData"
         class="container mw-920 mt-8rem"
-        :class="{ 'mb-3rem': !(article.cta && article.cta.hidden) }"
+        :class="{ 'mb-3rem': !(blogData?.cta && blogData.cta.hidden) }"
```

Comment:
After having `v-if` do we still need `?`for `blogData?.cta` etc?

Reply:
I think it's fine

Reply:
I see! Also, I think, the `v-if`  will eliminate the need of using `?.`

Reply:
Anyway, it's okay for now!